canvas_account_admin_tools/settings/base.py

```diff
@@ -88,14 +89,19 @@
     # 'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'django.middleware.security.SecurityMiddleware',
     'django.middleware.common.CommonMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
 ]
 
 FORM_RENDERER = 'djng.forms.renderers.DjangoAngularBootstrap3Templates'
 
 AUTHENTICATION_BACKENDS = [
     'django_auth_lti.backends.LTIAuthBackend',
+    'django.contrib.auth.backends.ModelBackend',
+    'harvardkey_cas.backends.CASAuthBackend',
 ]
 
+#CAS_SERVER_URL = 'https://www.pin1.harvard.edu/cas/'
```

Comment:
This should be uncommented, right?

Reply:
Oh -- glad you caught that. I had commented that as an experiment. Things seemed to be working properly with it commented, but I had meant to ask about it.

Reply:
will remove line completely subsequent to investigation by Chris/Colin